handler/handler.go

```diff
@@ -94,8 +94,14 @@ func (h handler) redirect(w http.ResponseWriter, r *http.Request) {
 
 	model, err := h.storage.Load(code)
 	if err != nil {
-		w.WriteHeader(http.StatusNotFound)
-		w.Write([]byte("URL Not Found"))
+		data, err := ioutil.ReadFile("frontend/index.html")
```

Comment:
It's better to use there [http.ServeFile()](https://golang.org/pkg/net/http/#ServeFile). So you don't need to handle errors etc.

Reply:
Huh! I didn't know about that... thanks for pointing that out. I will check this one also.